src/app/panels/image-panel/image-panel.component.html

```diff
@@ -1,14 +1,34 @@
-<evt-panel
-    [hideHeader]="true"
-    [hideFooter]="true">
+<evt-panel [showSecondaryContent]="isSecondaryContentOpened()">
     <div header-left>
-        <!-- TODO: Add dropdowns for navigation -->
+    <ng-select 
+        class="mr-1"
+        bindLabel="label"
+        bindValue="label"
+        [searchable]="true" 
+        [clearable]="false"
+        [ngModel]="currentImg"
+        (change)="toggleImg($event)">
+        <ng-option [value]="item" *ngFor="let item of images;">
+            {{ item.label }}
+        </ng-option>
+    </ng-select>
+    <evt-ms-desc-selector (selectionChange)="setMsDescID($event)" (msDescOpen)="isMsDescOpen($event)"></evt-ms-desc-selector>
     </div>
     <div content>
         <evt-osd
             *ngIf="manifest"
-            [manifestURL]="manifest">
+            [manifestURL]="manifest"
+            [page]="imageIndex"
+            (data)="eventHandler($event)"
```

Comment:
Since you are vinding to page change event use that instead.

Reply:
page change event does not consider the page change through arrows

Reply:
make the buttons fire that event instead